elisp/mew-pop.el

```diff
@@ -39,7 +39,7 @@
 ;;;
 
 (defvar mew-pop-fsm
-  '(("greeting"      nil ("\\+OK" . "capa"))
+  '(("greeting"      t   ("\\+OK" . "capa"))
```

Comment:
This changes "greeting" from a single line to multiple lines.
If the source of problem is "CAPA", this change should have no effect.

Reply:
When using GnuTLS, POP greetings are not received.
So a dummy CAPA is sent and the response is used as a substitute for the greeting.
https://github.com/kazu-yamamoto/Mew/blob/d41dc93785d231f1e391ba61893aacd1331d5726/elisp/mew-pop.el#L777